src/utils/components/InstanceTypeModal/utils/util.ts

```diff
@@ -1,28 +1,22 @@
 import { parseSize } from 'xbytes';
 
-import { InstanceTypeSize } from '@catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/utils/types';
 import {
   INSTANCETYPE_CLASS_DISPLAY_NAME,
   INSTANCETYPE_DESCRIPTION_ANNOTATION,
   REDHAT_COM,
 } from '@kubevirt-utils/components/AddBootableVolumeModal/components/VolumeMetadata/components/InstanceTypeDrilldownSelect/utils/constants';
 import { VENDOR_LABEL } from '@kubevirt-utils/constants/constants';
 import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { InstanceTypeSeries, InstanceTypeSize } from '@kubevirt-utils/resources/instancetype/types';
 import { getAnnotation, getLabel } from '@kubevirt-utils/resources/shared';
 import { InstanceTypeUnion } from '@virtualmachines/details/tabs/configuration/utils/types';
 
-import {
-  InstanceTypeRecord,
-  InstanceTypesSeries,
-  InstanceTypesSizes,
-  MappedInstanceTypes,
-} from './types';
-
-export const getInstanceTypeItemSizePrettyDisplay = (it: InstanceTypeUnion): string =>
-  `${it?.metadata.name.split('.').pop()}: ${it?.spec?.cpu?.guest} ${t('CPUs')}, ${
-    it?.spec?.memory?.guest
-  } ${t('Memory')}`;
+import { InstanceTypeRecord, MappedInstanceTypes } from './types';
 
+export const getInstanceTypeItemSizePrettyDisplay = (it: InstanceTypeUnion): string => {
+  const name = it?.metadata?.name?.split('.').pop() ?? it?.metadata?.name ?? '';
```

Comment:
If we add `K8sResourceCommon` to the `InstanceTypeUnion` type you can use the `getName` selector, which will make this look a little cleaner.

It would also be nice to add selectors to src/utils/resources/instancetype for the 'guest' and 'memory' properties below.